chore: update examples to use `uv` (#2624)
pallets-eco/flask-admin: examples/babel/README.md

# Flask-Babel Integration Example

This example show how to translate Flask-Admin into different language using customized version of the `Flask-Babel <https://github.com/python-babel/flask-babel>`

## How to run this example

Clone the repository and navigate to this example:

```bash
git clone https://github.com/pallets-eco/flask-admin.git
cd flask-admin/examples/babel
```

> This example uses [`uv`](https://docs.astral.sh/uv/) to manage its dependencies and developer environment.

> To install the minimum version of Python this example supports, create your virtual environment, and install the required dependencies, run:
> 
> ```bash
> uv sync
> ```
> 
> Run the application:
> 
> ```bash
> uv run main.py
> ```

Reviewer: ```suggestion
Run the example using `uv`, which will manage the environment and dependencies automatically:

```bash
uv run main.py
```
```
